eng/templates/azure-pipelines-shared-jobs.yml

```diff
@@ -0,0 +1,134 @@
+parameters:
+- name: isOfficial
+  displayName: Indicates whether to use official or unofficial 1ES templates
+  type: boolean
+  
+stages:
+  - stage: Build
+    dependsOn: []
+    displayName: Build
+    jobs:
+    - template: /eng/common/templates-official/jobs/jobs.yml@self
+      parameters:
+        artifacts:
+          publish:
+            logs: true
+            ${{ if in(variables['Build.SourceBranch'], 'refs/heads/main', 'refs/heads/production') }}:
+              manifests: true
+        enableTelemetry: true
+        enableMicrobuild: false
+        enablePublishTestResults: false
+        enablePublishUsingPipelines: ${{ variables._PublishUsingPipelines }}
+        jobs:
+        - job: Windows_NT
+          timeoutInMinutes: 90
+          displayName: Build repository
+          pool:
+            name: $(DncEngInternalBuildPool)
+            image: 1es-windows-2022
```

Comment:
In the main YAML we reference `1es-windows-2019` but here we use the same VM image just updated. Can we maybe use `2022` in the main YAML and remove this `pool` section?